bobweb/bob/openai_api_utils.py

```diff
@@ -83,13 +83,13 @@ def msg_serializer_for_vision_models(message: GptChatMessage) -> dict[str, str]:
     return {'role': message.role.value, 'content': content}
 
 
-gpt_3_16k = GptModel(
-    name='gpt-3.5-turbo-0125',
-    major_version=3,
+gpt_4o_mini = GptModel(
+    name='gpt-4o-mini',
+    major_version=4,
     has_vision_capabilities=False,
```

Comment:
```suggestion
    has_vision_capabilities=True,
```
Toi 4o-mini tukee myös visionia apin kautta.